packages/editor-ui/src/components/NodeExecuteButton.vue

```diff
@@ -12,6 +12,7 @@
 					:label="buttonLabel"
 					:type="type"
 					:size="size"
+					:icon="isFormTriggerNode && 'flask'"
```

Comment:
can we roll this out globally? maybe we can just roll this out globally already? https://linear.app/n8n/issue/ADO-129/update-copy-execute-test

Reply:
this is up to product to decide